README.md

```diff
@@ -47,14 +47,16 @@ npm install cisco-telescope
 
 > Cisco OpenTelemetry Distribution is activated and instruments the supported libraries once the module is imported.
 
+To initizlize the library, you'll need a Telescope Token, which is taken from your [Account tab on the Telescope console Settings page](https://console.telescope.app/settings/account).
+
 #### javascript
 
 ```javascript
 const { ciscoTracing } = require('cisco-telescope');
 
 const userOptions = {
   serviceName: 'my-app-name',
-  ciscoToken: 'cisco-token',
+  ciscoToken: 'telescope-token',
```

Comment:
it's more important to change the key rather than the example value in my opinion